lib/reporters/cli/index.js

```diff
@@ -81,9 +81,20 @@ PostmanCLIReporter = function (emitter, reporterOptions, options) {
         // for some reason, if there is no run summary, it is unexpected and hence don't validate this
         var run = this.summary.run;
 
+        // show the body of the response for single request
+        // eslint-disable-next-line max-len
+        if (options.singleRequest && run.executions && run.executions.length && run.executions[0].response && run.executions[0].response.text()) {
+            print(run.executions[0].response.text());
```

Comment:
@raghavbhatia336 
- Can we improve the output for the response text? Maybe we can provide a flag on basis of which we will prettify the output or not.
- Can we verify the output of different formats like XML, or blob? How is it looking in the CLI?
- What would happen for very long outputs?